client/src/components/modal/Modal.js

```diff
@@ -0,0 +1,75 @@
+/* MODAL CPMPONENT
+- Can be utilized to render complete pages or syimpke on the fly dialogs, whatever is passed as implicit children arg
+
+  <Modal>
+    <WhatEver/>
+  </Modal>
+
+- If the Modal is used in combination with a Route (via the 'render'-attr / prop) and a path is provided, the rendered Modal receives its own visible url - e.g.:
+
+      <Route
+        path="/newCommunity"
+        render={() => {
+          return (
+            <Modal>
+              <NewCommunityModalPage />
+            </Modal>
+          );
+        }}
+      />
+*/
```

Comment:
Well, you know what I would comment here 😆
Ok I stop here and don't comment on unused code in other files.

Reply:
Yeah - bad old habit of mine - I leave ode comments hanging around as a reminder for stuff yet to be done etc. - but it's a bad practice, I know - in my defense: It was very late - plus, in the above case:  It's more a documentation on how to use that function / component...

Reply:
Ah ok, for documentation it makes sense to leave it in. 
Yeah I remember you talked about this habit or way of working before:)

Reply:
If you have just a tool, to create some components in isolation............................([storybook](https://storybook.js.org/)) ;-) Boris posted the video about how to set it up and work with in slack.

Reply:
Will have a look into storybook - sounds awesome